happy:
  BrowsU_C: 1
  BrowsU_L: 1
  BrowsU_R: 1
  JawOpen: 0.3
  MouthSmile_L: 1
  MouthSmile_R: 1
  MouthDimple_L: 0.4
  MouthDimple_R: 0.4
  LipsUpperOpen: 0.3
  LipsLowerOpen: 0.63
  CheekSquint_L: 1
  CheekSquint_R: 1

sad:
  BrowsD_L: 1
  BrowsD_R: 1
  BrowsU_C: 1
  JawOpen: 0.16
  MouthFrown_L: 1
  MouthFrown_R: 1
  LipsLowerClose: 1
  ChinLowerRaise: 1
  ChinUpperRaise: 0.5
  CheekSquint_L: 1
  CheekSquint_R: 1

surprised:
  EyeOpen_L: 1
  EyeOpen_R: 1
  BrowsU_C: 1
  BrowsU_L: 1
  BrowsU_R: 1
  JawOpen: 1
  JawChew: 1
  LipsLowerDown: 1

evil:
  EyeSquint_L: 1
  EyeSquint_R: 1
  BrowsD_L: 1
  BrowsD_R: 1
  JawOpen: 0.07
  MouthLeft: 0.56
  MouthSmile_L: 1
  MouthSmile_R: 1
  MouthDimple_L: 1
  LipsUpperUp: 0.3
  LipsLowerDown: 1
  LipsUpperOpen: 0.85

afraid:
  BrowsD_L: 0.6
  BrowsD_R: 0.6
  BrowsU_C: 1
  JawOpen: 0.15
  MouthFrown_L: 1
  MouthFrown_R: 1
  LipsLowerOpen: 0.32
  ChinLowerRaise: 1
  ChinUpperRaise: 0.84
  CheekSquint_L: 1
  CheekSquint_R: 1

horrified:
  EyeOpen_L: 1
  EyeOpen_R: 1
  BrowsU_C: 1
  BrowsU_L: 1
  BrowsU_R: 1
  JawOpen: 0.7
  MouthFrown_L: 1
  MouthFrown_R: 1
  LipsLowerClose: 1
  LipsUpperUp: 0.3
  ChinLowerRaise: 0.6
  ChinUpperRaise: 1
  CheekSquint_L: 1
  CheekSquint_R: 1

annoyed:
  EyeOpen_L: 0.5
  EyeOpen_R: 0.5
  BrowsD_L: 1
  BrowsD_R: 1
  JawOpen: 0.1
  MouthFrown_L: 1
  MouthFrown_R: 1
  LipsLowerClose: 0.8
  LipsUpperUp: 0.13
  ChinLowerRaise: 1
  ChinUpperRaise: 1

innocent:
  BrowsU_C: 1
  MouthSmile_L: 1
  MouthSmile_R: 1
  MouthDimple_L: 0.5
  MouthDimple_R: 0.5
  LipsLowerOpen: 0.35

violent:
  EyeOpen_L: 1
  EyeOpen_R: 1
  BrowsD_L: 1
  BrowsD_R: 1
  BrowsU_L: 0.5
  BrowsU_R: 0.5
  MouthFrown_L: 1
  MouthFrown_R: 1
  LipsStretch_L: 1
  LipsStretch_R: 1
  LipsUpperUp: 0.8
  LipsLowerOpen: 0.3
  ChinUpperRaise: 0.7
  Sneer: 1
  CheekSquint_L: 1
  CheekSquint_R: 1

eureka:
  EyeSquint_L: 1
  EyeSquint_R: 1
  EyeOpen_L: 1
  EyeOpen_R: 1
  BrowsU_C: 1
  JawOpen: 0.7
  MoutSmile_L: 1
  MouthSmile_R: 1
  LipsUpperUp: 0.5
  LipsLowerDown: 0.5
  CheekSquint_L: 1
  CheekSquint_R: 1
